src/components/BillingRemaining.tsx

```diff
@@ -0,0 +1,144 @@
+import { Spinner } from '@src/components/Spinner'
+import { useSelector } from '@xstate/react'
+import { useState } from 'react'
+
+import { CustomIcon } from '@src/components/CustomIcon'
+import type { BillingActor } from '@src/machines/billingMachine'
+
+export enum BillingRemainingMode {
+  ProgressBarFixed,
+  ProgressBarStretch,
+}
+
+export interface BillingRemainingProps {
+  mode: BillingRemainingMode
+  billingActor: BillingActor
+}
+
+const Error = (props: { error: Error }) => {
+  const [showMessage, setShowMessage] = useState(false)
+
+  const fadedBg = 'rgba(127, 127, 127, 0.2)'
+  const fadedFg = 'rgba(255, 255, 255, 0.6)'
```

Comment:
```suggestion
  const fadedBg = 'rgba(127, 127, 127, 1)'
  const fadedFg = 'rgba(255, 255, 255, 1)'
```

Just make these opaque, they don't have a reason to be transparent.